config/software/erlang.rb

```diff
@@ -60,7 +60,10 @@
   )
   env.delete("CPPFLAGS")
 
-  env.delete("TYPE")
+  # The TYPE env var sets the type of emulator you want
+  # We want the default so we delete TYPE from ENV in case
+  # it was set by CI.
+  ENV.delete("TYPE")
```

Comment:
For other reviewers, this is more correct because our underlying shellout libraries append the passed environment to the inherited environment so we need to remove it from the inherited environment.  

This could have a side-effect on other plans since it modifies the environment for this process; but, I doubt we have any other plans that rely on this.  An alternative could be

```ruby
env["TYPE"] = ""
```

But, let's go with this for now since @jeremiahsnapp has tested it already.